homeassistant/components/cloud/__init__.py

```diff
@@ -46,7 +51,8 @@
         vol.Optional(CONF_USER_POOL_ID): str,
         vol.Optional(CONF_REGION): str,
         vol.Optional(CONF_RELAYER): str,
-        vol.Optional(CONF_ALEXA): ALEXA_SCHEMA
+        vol.Optional(CONF_ALEXA): ASSISTANT_SCHEMA,
+        vol.Optional(CONF_GOOGLE_ASSISTANT): ASSISTANT_SCHEMA,
```

Comment:
I like this. Maybe even more better as the `hass.info`. Maybe we should _later_ at this to compoment config and store that on `setup` into `hass.data` so that we can use this here.

Reply:
Yeah, I realized we did the same for Z-Wave. I'll open PRs to do this for Alexa and Google Assistant too